airmozilla/popcorn/views.py

```diff
@@ -51,11 +42,68 @@ def event_meta_data(request):
 
     assert response.status_code == 302, response.status_code
 
+    purl = urlparse(response.headers['location'])
+    location = r'{0}://{1}{2}'.format(purl.scheme, purl.hostname, purl.path)
+
+    return location
+
+
+def get_thumbnail(event):
+    image = event.picture and event.picture.file or event.placeholder_img
+    geometry = '160x90'
+    crop = 'center'
+
+    return thumbnail(image, geometry, crop=crop)
+
+
+@add_cors_header('*')
+@json_view
+def event_meta_data(request):
+    slug = request.GET.get('slug')
+    event = get_object_or_404(Event, slug=slug)
+
+    video_url = get_video_url(event)
+
+    thumb = get_thumbnail(event)
+
     return {
         'title': event.title,
         'description': event.short_description or event.description,
```

Comment:
It will just ellipse after a certain amount of characters. HTML tags should be handled by the js template engine